pkg/daemon/config.go

```diff
@@ -107,43 +107,45 @@ func tryToLoadOldConfig(nodeProfilesJson []byte) ([]ptpv1.PtpProfile, bool) {
 }
 
 func (output *ptp4lConf) populatePtp4lConf(config *string) error {
-	lines := strings.Split(*config, "\n")
 	var currentSectionName string
 	var currentSection ptp4lConfSection
 	output.sections = make([]ptp4lConfSection, 0)
 	globalIsDefined := false
 
-	for _, line := range lines {
-		line = strings.TrimSpace(line)
-		if strings.HasPrefix(line, "#") {
-			continue
-		} else if strings.HasPrefix(line, "[") {
-			if currentSectionName != "" {
-				output.sections = append(output.sections, currentSection)
+	if config != nil {
+		for _, line := range strings.Split(*config, "\n") {
+			glog.Infof(line)
```

Comment:
Is this log temporary? If not then need to add a context log explaining what this is.

Reply:
Not needed.  will remove.